app/Policies/ChatMessagesPolicy.php

```
@@ -0,0 +1,47 @@
+    public function view(User $user, ChatMessage $chatMessage): bool
+    {
+        /** @var Chat|null $chat */
+        $chat = $chatMessage->chat;
+
+        return $chat?->users()->where('users.id', $user->getKey())->exists() ?? false;
+    }
+
+    /**
+     * Determine whether the user can create models.
+     */
+    public function create(User $user): bool
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$user'. (undefined)

(UnusedFormalParameter)